app/(items)/artist/[artist-id]/page.tsx

```diff
@@ -14,33 +16,66 @@ export default async function ArtistPage({ params }: ArtistPageProps) {
     id: artistId,
   })
 
+  const { tracks } = await getArtistTopTracks({ id: artistId })
```

Comment:
How about renaming `artistId` variable to `id`? It will make the code simpler and since this is artist page you already know that param id will be an artist id.
```suggestion
  const { tracks } = await getArtistTopTracks({ id })
```